src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -306,7 +316,12 @@ public FeedbackResponseCommentAttributes updateFeedbackResponseComment(FeedbackR
             throw new EntityDoesNotExistException(ERROR_UPDATE_NON_EXISTENT + newAttributes.toString());
         }
         
-        frc.setGiverEmail(newAttributes.giverEmail);
+        if (isNewComment) {
+            frc.setGiverEmail(newAttributes.giverEmail);     
+        } else {
+            frc.setLastEditorEmail(newAttributes.giverEmail);
```

Comment:
why need a branch here? this branch results in many changes that I dont get the purpose